src/librustc_data_structures/bitvec.rs

```diff
@@ -47,7 +47,11 @@ impl<C: Idx> BitVector<C> {
     #[inline]
     pub fn contains(&self, bit: C) -> bool {
         let (word, mask) = word_mask(bit);
-        (self.data[word] & mask) != 0
+        if let Some(word) = self.data.get(word) {
```

Comment:
Tiny Nit (feel free to ignore): this could be written as `self.data.get(word).map_or(false, |word| (word & mask) != 0)

Reply:
`map_or` etc usually makes me more confused then seeing code like this so I prefer this in general when not in a conditional or other space-constrained place.

Reply:
That's completely reasonable =)